src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -786,9 +799,6 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
             @chmod($dir.$file, 0666 & ~umask());
         }
 
-        // track changes made to the container directory
-        $container->fileExists(dirname($dir.$file));
```

Comment:
Removes a transient test (`CacheClearCommandTest::testCacheIsFreshAfterCacheClearedWithWarmup: Meta file "FixtureTestDebugProjectContainer.php.meta" is not fresh`)
and allows debugging the container by patching its generated when needed (I added this code in 3.4. A bad idea.)